octue/log_handlers.py

```diff
@@ -3,11 +3,17 @@
 import os
 from urllib.parse import urlparse
 
+from octue.utils.colour import colourise
+
 
 # Logging format for analysis runs. All handlers should use this logging format to make logs consistently parseable.
 LOG_RECORD_ATTRIBUTES_WITH_TIMESTAMP = ["%(asctime)s", "%(levelname)s", "%(name)s"]
 LOG_RECORD_ATTRIBUTES_WITHOUT_TIMESTAMP = LOG_RECORD_ATTRIBUTES_WITH_TIMESTAMP[1:]
 
+# "tab10" colours from Seaborn.
+COLOUR_PALETTE = ["1f77b4", "ff7f0e", "2ca02c", "d62728", "9467bd", "8c564b", "e377c2", "7f7f7f", "bcbd22", "17becf"]
```

Comment:
"tab10" is matplotlib default colours. Could you use the colorblind palette instead, please? I think it'll be both more aesthetically pleasing (it's somewhat more muted) and less disadvantaging to colourblind folks.